File: src/main/osu/beatmap/sections/BeatmapColor.java
package main.osu.beatmap.sections;

import java.util.ArrayList;

import main.osu.RGBColor;
import main.osu.beatmap.ComboColor;

/**
 * Contains information about combo and skin colors of a beatmap.
 */
public class BeatmapColor {
    /**
     * The combo colors of this beatmap.
     */
    public ArrayList<ComboColor> comboColors = new ArrayList<>();

    /**
     * The color of the slider border.
     */
    public RGBColor sliderBorderColor;
}

File: src/com/rian/osu/beatmap/sections/BeatmapColor.kt
package com.rian.osu.beatmap.sections

import com.reco1l.framework.*
import com.rian.osu.beatmap.ComboColor

/**
 * Contains information about combo and skin colors of a beatmap.
 */
class BeatmapColor {
    /**
     * The combo colors of this beatmap.
     */
    @JvmField
    val comboColors = mutableListOf<ComboColor>()

    /**
     * The color of the slider border.
     */
    var sliderBorderColor: ColorARGB? = null
}
